src/core/counter/tables.rs

```diff
@@ -0,0 +1,381 @@
+use crate::core::sizage::Sizage;
+use crate::error::{Error, Result};
+
+pub(crate) fn sizage(s: &str) -> Result<Sizage> {
+    Ok(match s {
+        "-A" => Sizage {
+            hs: 2,
+            ss: 2,
+            fs: 4,
+            ls: 0,
+        },
```

Comment:
This was way nicer before cargo fmt.. My reasoning was that I wanted to see the labels in the code.